Code-4-Community/lucys-love-bus-frontend
# modals

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

This PR adds the LoginModal component. This component is a modal which has three content pages: the login page, the forgot password page, and the reset confirmation page. This is a necessary frontend component.

## This PR

The changes are self-explanatory. A new component LoginModal is added that can be open from the /modals route.

## Screenshots
 
![Screenshot 2020-10-28 142048](https://user-images.githubusercontent.com/57075562/97498091-ce921b00-1928-11eb-985e-4d73649b258b.jpg)
![Screenshot 2020-10-28 142035](https://user-images.githubusercontent.com/57075562/97498092-ce921b00-1928-11eb-8c4b-06366eaf242d.jpg)
![Screenshot 2020-10-28 142023](https://user-images.githubusercontent.com/57075562/97498094-cf2ab180-1928-11eb-8796-4d89d057301d.jpg)

## Verification Steps

Thorough testing was done to ensure that the modal works as expected. Specifically, testing in local environment to see that all changes exhibited expected behavior.
